### PR TITLE
Group routes

### PR DESCRIPTION
**Pull request by Marielvacacruz**
Was able to return the correct object but the numMembers attribute (aggregate count) is off by 1. I tested the regular GET ALL GROUPS route and that is returning the correct numMembers count so I'm not sure why it is not allowing this route to do so

### CODE REVIEW

File: authenticate-me/backend/routes/groups.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            {
+                model: Image,
+                attributes: []
+            },
+            {
+                model: Member,
+                attributes: []
+            },
+            {
+                model:User //can't figure out how to alias as 'Organizer'
```
Comment (WJVincent) [inline authenticate-me/backend/routes/groups.js]: as: "Organizer", you will also need to add the alias to the association in your group model

**Review by WJVincent — CHANGES_REQUESTED** [review]
Other than setting up the User alias as Organizer for Group looks good

File: authenticate-me/backend/routes/groups.js (hunk 1 above)
Comment (Marielvacacruz) [reply authenticate-me/backend/routes/groups.js]: ok yeah that's the only solution I had found, so I'll update that now

**Review by WJVincent — APPROVED** [review]
Looks good

**Pull request merged**